README.md

```diff
@@ -0,0 +1,209 @@
+//  {Name: "Grace Recipient", Address: "grace.recipient@example.com"},
+// }
+```
+
+get custom headers:
+
+```go
+email.Headers.ExtraHeaders
+// map[string][]string{
+//    "X-Clacks-Overhead": {"GNU Terry Pratchett"},
+// }
+```
+
+get decoded bodies:
+
+```go
+email.Text
+// "The quick brown fox jumps over a lazy dog..."
+
+email.HTML
+// "<html><div dir="ltr"><p>The quick brown fox jumps over a lazy dog..."
+```
+
+inline files:
+
+```go
+email.InlineFiles
+// []InlineFile{
+//    {
+//        ContentType: ContentTypeHeader{
+//            ContentType: "image/jpeg",
+//            Params: map[string]string{
+//                "name": "inline-jpg-image-without-disposition.jpg",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: "",
+//            Params:             map[string]string(nil),
+//        },
+//        Data: []byte{255, ...},
+//    },
+//    {
+//        ContentID: "inline-jpg-image.jpg@example.com",
+//        ContentType: ContentTypeHeader{
+//            ContentType: "image/jpeg",
+//            Params: map[string]string{
+//                "name": "inline-jpg-image-name.jpg",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: inline,
+//            Params: map[string]string{
+//                "filename": "inline-jpg-image-filename.jpg",
+//            },
+//        },
+//        Data: []byte{255, ...},
+//    },
+// }
+```
+
+and attached files:
+
+```go
+email.AttachedFiles
+// []AttachedFile{
+//    {
+//        ContentType: ContentTypeHeader{
+//            ContentType: "application/pdf",
+//            Params: map[string]string{
+//                "name": "attached-pdf-name.pdf",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: attachment,
+//            Params: map[string]string{
+//                "filename": "attached-pdf-filename.pdf",
+//            },
+//        },
+//        Data: []byte{37, ...},
+//    },
+//    {
+//        ContentType: ContentTypeHeader{
+//            ContentType: "application/pdf",
+//            Params: map[string]string{
+//                "name": "attached-pdf-without-disposition.pdf",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: "",
+//            Params:             map[string]string(nil),
+//        },
+//        Data: []byte{37, ...},
+// }
+```
+
+The same parser and methods will work for other languages, text encodings, 
+and transfer encodings:
+
+```go
+r := strings.NewReader(```Subject: =?ISO-2022-JP?Q?=1B=24=42=24=24=24=6D=24=4F=32=4E=1B=28=42?=
+Content-Type: text/plain; charset=ISO-2022-JP
+
+
+=1B$B?'$OFw$($I=1B(B
+=1B$B;6$j$L$k$r=1B(B```)
+
+email, _ := letters.ParseEmail(r)
+
+email.Headers.Subject
+// "いろは歌"
+
+email.Text
+// "色は匂えど散りぬるを..."
+```
+
+## Current Scope and Features
```

Comment:
What do you think about moving this part above the `Quickstart` section?
It is somehow related to the thread explaining to me what it is all about.

Of course, as an engineer, I would like to have instructions on how to use it almost at the top, but I wonder if this order could be beneficial for users:

1. Letters, or how to parse emails in Go
2. Current Scope and Features
3. Quickstart